tools/ScrollAnimationAssembler/apply.dm

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
i know that dmdoc won't work here but still would be really nice if you could doc what this proc does